swap_meet/item.py

```diff
@@ -1,2 +1,45 @@
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self,category = "", condition = None):      
+        self.category = category
+        self.condition = condition
+        if condition is None:
+           self.condition = 0
+         
+
+    def __str__(self):      
+        return f"Hello World!"
+
+    def condition_description(self):
```

Comment:
Good job making an if/elif statement here. I would look into how you could possibly improve this block of code by using other data structures.